decidim-core/app/views/decidim/pages/_tabbed.html.erb

```diff
@@ -1,45 +1,38 @@
-<div class="wrapper">
-  <div class="row column">
-    <div class="page-title-wrapper">
-      <h1 class="heading1 page-title">
+<%# NOTE: this page does not use a regular layout %>
+<div class="page__layout">
+  <div>
+
+    <div class="text-center">
+      <h1 class="h1 decorator inline-block mb-12">
         <%= title %>
       </h1>
-      <p class="intro--text subheading1">
+      <p class="text-lg text-gray-2">
         <%= description %>
       </p>
     </div>
 
-    <div class="main-container tabbed-container">
-      <%= cell "decidim/tos_page", :announcement %>
+    <%= cell "decidim/tos_page", :announcement %>
 
-      <div class="row collapse main-container--side-panel">
-        <div class="columns medium-4 large-3">
-          <div class="side-panel">
-            <ul class="tabs vertical side-panel__tabs">
-              <% pages.each do |sibling| %>
-                <li class="tabs-title <%= page == sibling ? "is-active" : nil %>">
-                  <%= link_to translated_attribute(sibling.title), page_path(sibling.slug) %>
-                </li>
-              <% end %>
-            </ul>
-          </div>
-        </div>
+    <div class="page__tabbed">
+      <div>
+        <ul class="page__tabbed-list">
```

Comment:
I find this long list of pages problematic on mobile views as it takes the most of the screen estate on this page. Look at this for example:
![Tabbed pages on mobile](https://user-images.githubusercontent.com/864340/184156333-491adea0-69eb-4acf-a13c-7f0b87c6009f.png)

1. What is the main point on this page? Is it the navigation or the text?
2. Which takes most screen estate of this page?
3. How long does the user have to scroll before reaching the main point of this page?

E.g. in the Helsinki instance we have solved this by hiding this under a navigation bar on mobile as follows:
![Page navigation](https://user-images.githubusercontent.com/864340/184156736-6c566610-2a30-466e-aa9e-7f7d3799b7fc.png)

You can see it here:
https://omastadi.hel.fi/pages?locale=en

Don't mind the extraordinary big header element there, it was not my decision. I left it out purposefully from the screenshot.

Reply:
I've tried to give this a thought, but I think we'd need @rober-gd here. I cannot achieve it with no code duplication or additional resources. Meanwhile, I'll address it in the pending issue

Reply:
This component should be the same as `/account`

Reply:
This issue was moved to #9753.